### ISSUE TITLE
Logo in email alerts looks out of place

### ISSUE THREAD

**Issue by henare** [openaustralia/morph#624]
Related #556.

**Comment by mlandauer** [comment]
I agree. As we have gotten rid of it on our landing page we should get rid of it on the email alerts

**Comment by mlandauer** [comment]
I changed the title so that it's about more than just removing the logo. It's about doing a bit of refresh to make it more consistent with the rest of the site

**Comment by equivalentideas** [comment]
Some thoughts on this:

I personally really like the very simple design of the Github notification emails. No added boxes or borders, just text. It's also very flexible to different devices and clients.

But maybe that's no appropriate as this is a bit more of an alert that we want people to feel compelled to respond to. Not sure.

I think keeping it simple, and being very deliberate with the visual elements we introduce. But of course we'd do that anyway ;-)

**Comment by mlandauer** [comment]
I think like the title of the issue suggests that we should make the look of the email alerts similar to the look of the app and how the "scrapers that need your attention" or shown on the landing page.

Also, we're showing some complex information including things like logs which require strong visual separation. An email alert with just text isn't going to work.

The Github notification emails work as plain text because they're showing information about one thing. e.g. this person has added this comment.

Our email alerts are considerably more complicated than that.

**Comment by equivalentideas** [comment]
So here's the content in the current design:

![screen shot 2015-05-22 at 2 44 13 pm](https://cloud.githubusercontent.com/assets/1239550/7764224/3b50aede-0091-11e5-875a-a4e8c03a99c2.png)

And we want to make it more consistent with the app using the alert style on the landing page:

![screen shot 2015-05-22 at 2 36 53 pm](https://cloud.githubusercontent.com/assets/1239550/7764225/3fae5242-0091-11e5-9672-fe34df46378f.png)

**Comment by equivalentideas** [comment]
As this issue is just about the look I'm assuming we want to keep the content the same. Here's one way the log content could be added to the alert style:

![screen shot 2015-05-22 at 3 03 09 pm](https://cloud.githubusercontent.com/assets/1239550/7764367/b3e39b3e-0093-11e5-8654-0d4d29640340.png)

We could also drop off the elements from the elements from the righthand side like the language and still keep the same feel. Is that important for the user at this point? Adding that is also a change in the content.

**Comment by benrfairless** [comment]
Thanks for raising this, and sorry it has sat without a response for so long.

We are tidying up the morph.io issue tracker. This issue dates from 2015 and, on review, looks to have gone stale (the idea has been overtaken, superseded, or not picked up in the years since), so we are closing it as part of a backlog review to keep the tracker focused on work we can realistically act on. Closing it is not a judgement that the underlying idea was wrong.

If this is still relevant to you, please reopen it or open a fresh issue with up to date detail and we will gladly take another look.

This triage was carried out with AI assistance (Claude Opus 4.8 via Claude Code) and reviewed by the OpenAustralia Foundation team.
